__tests__/app.test.js

```diff
@@ -0,0 +1,26 @@
+const request = require('supertest')
+const seed = require('../db/seeds/seed')
+const app = require('../app')
+const db = require('../db/connection')
+const testData = require('../db/data/test-data')
+
+
+beforeEach(()=>seed(testData))
+
+afterAll(()=>{
+    if(db.end) db.end();
+})
+
+describe('1. GET/api/topics', ()=>{
+    test('status 200, should respond with topics objects in array',()=>{
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((response)=>{
+            expect(response.body.topics).toEqual(expect.any(Array));
+            expect(Object.keys(response.body.topics[0])).toEqual(
```

Comment:
Are you only expecting a single object back from this endpoint? This assertion should test that all topics have the same keys and testing for what type the value of these are. Looking into .toMatchObject() will help in this case.